client/App.jsx

```diff
@@ -22,16 +53,18 @@ class App extends React.Component {
       totalScore: this.state.totalScore,
       totalReviews: this.state.totalReviews,
       sortedValue: event.target.value
+      reviewsArray: this.state.reviewsArray,
+      showReviews: this.state.showReviews
     })
   }
 
   render() {
     return (
       <div id="container">
-        <h2 id="componentTitle" >Customer Reviews</h2>
-        <Stats value={this.state.value} handleChange={this.handleChange.bind(this)}/>
-        <ReviewsFeed />
-        <button id="loadMoreButton">Load More</button>
+        <h2 id="componentTitle">Customer Reviews</h2>
+        <Stats />
+        <ReviewsFeed reviewsArray={this.state.reviewsArray} showReviews={this.state.showReviews}/>
+        <button id="loadMoreButton" onClick={this.loadMore.bind(this)}>Load More</button>
```

Comment:
Typically, cross-component functions are bound at the end of the constructor instead of in-line, though it's clearly a readability thing rather than an actual functionality one.